packages/esm-service-queues-app/src/patient-search/visit-form/existing-visit-form.component.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback, useState } from 'react';
+import { Button, ButtonSet, Form, Row, Stack } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import {
+  type ConfigObject,
+  ExtensionSlot,
+  showSnackbar,
+  useConfig,
+  useLayoutType,
+  type Visit,
+} from '@openmrs/esm-framework';
+import { addQueueEntry, useVisitQueueEntries } from '../../active-visits/active-visits-table.resource';
+import { type SearchTypes } from '../../types';
+import styles from './visit-form.scss';
+
+interface ExistingVisitFormProps {
+  toggleSearchType: (searchMode: SearchTypes, patientUuid) => void;
+  visit: Visit;
+  closePanel: () => void;
+}
+
+const ExistingVisitForm: React.FC<ExistingVisitFormProps> = ({ visit, toggleSearchType, closePanel }) => {
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const config = useConfig() as ConfigObject;
+  const visitQueueNumberAttributeUuid = config.visitQueueNumberAttributeUuid;
+  const { mutate } = useVisitQueueEntries('', '');
+
+  if (!visit) {
+    return null;
+  }
+
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+
+      // retrieve values from queue extension
+      const queueLocation = event?.target['queueLocation']?.value;
+      const serviceUuid = event?.target['service']?.value;
+      const priority = event?.target['priority']?.value;
+      const status = event?.target['status']?.value;
+      const sortWeight = event?.target['sortWeight']?.value;
+
+      setIsSubmitting(true);
+
+      addQueueEntry(
+        visit.uuid,
+        serviceUuid,
+        visit.patient.uuid,
+        priority,
+        status,
+        sortWeight,
+        queueLocation,
+        visitQueueNumberAttributeUuid,
+      ).then(
+        ({ status }) => {
+          if (status === 201) {
+            showSnackbar({
+              kind: 'success',
+              isLowContrast: true,
+              title: t('startAVisit', 'Start a visit'),
```

Comment:
You can change this label to reflect the queue operation

Reply:
Thanks @ojwanganto - looks like this PR was merged already by @makombe .  I tried to apply this change last night, but am running into build errors at the moment that is preventing me from committing.  @makombe maybe you can fix this for me since you merged the PR?  Otherwise, I'll address it in a follow-up commit once I have my environment back.